### PR TITLE
Update docs via `debops-optimize`

### PR DESCRIPTION
**Pull request by ganto**
Sorry, is a trivial PR...

Actually I wanted to implement support for LVM thin provisioning but had to learn, that this is not implemented yet upstream. Since I already had this beauty patch, I'll send it anyway.

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -7,29 +7,29 @@ v0.1.5
 *Released: 2016-05-23*
 
 - Sort configuration file sections to generate stable order of sections each
-  time. This should fix an issue where consequent runs of ``debops.lvm`` role
+  time. This should fix an issue where consequent runs of debops.lvm_ role
```
Comment (drybjed) [inline CHANGES.rst]: You should change this back to the old syntax for now, can be fixed later when the global reference file is updated.

File: docs/getting-started.rst [hunk 2]
```diff
@@ -26,7 +26,7 @@ and add your own host groups to it::
 Example playbook
 ----------------
 
-Here's an example playbook which uses ``debops.lvm`` role::
+Here's an example playbook which uses debops.lvm_ role::
```
Comment (drybjed) [inline docs/getting-started.rst]: This can be fixed later, it should be reverted for now.

**Comment by ganto** [comment]
Ok, thanks for reviewing

**Comment by drybjed** [comment]
@ganto You didn't need to close this PR. You could just reopen it and push your changes to the same branch in the repository.

**Pull request closed without merging**